### PR TITLE
Dart: drop formatting and clean up code

### PR DESCRIPTION
**Pull request by robertbastian**

### CODE REVIEW

File: tool/src/dart/mod.rs [hunk 1]
```diff
@@ -1,50 +1,933 @@
-mod class;
-mod formatter;
-
 use crate::common::{ErrorStore, FileMap};
+use askama::Template;
```
Comment (Manishearth) [inline tool/src/dart/mod.rs:2]: what changed in this file? Why is this diff so big?

Comment (robertbastian) [reply tool/src/dart/mod.rs:2]: oh I merged `class.rs` into it because the split was kind of arbitrary and copied from cpp2. deleted now

**Comment by sffc** [comment]
Will this fix https://github.com/unicode-org/icu4x/issues/4291?

File: tool/src/dart/mod.rs (hunk 1 above)
Comment (Manishearth) [reply tool/src/dart/mod.rs:2]: going to assume nothing else changed then

**Review by Manishearth — APPROVED** [review]

File: tool/src/dart/mod.rs (hunk 1 above)
Comment (robertbastian) [reply tool/src/dart/mod.rs:2]: Well there were some merging simplification changes, and changes to generated whitespace

**Pull request merged**